### DISCUSSION TITLE
Character limit in text-to-speech streaming

### DISCUSSION THREAD

**Discussion by nitin-airindia** [deepgram/community#990] [General help]
I was reading about Character Throughput Limits (https://developers.deepgram.com/docs/streaming-text-to-speech#character-throughput-limits). I'd a doubt regarding it, whether limit of 12k characters / 2 minutes applies to each websocket connection or it applies to my account (i.e. all websocket connections combined)?

**Comment by deepgram-community** [comment]
Hey there! It looks like you haven't connected your GitHub account to your Deepgram account. You can do this at https://community.deepgram.com - being verified through this process will allow our team to help you in a much more streamlined fashion.

**Comment by deepgram-community** [comment]
It looks like we're missing some important information to help debug your issue. Would you mind providing us with the following details in a reply?

- The programming language you are working in (e.g. JavaScript, Python).
- A request ID that triggered your error or issue.

**Comment by jkroll-deepgram** [comment]
Hi @nitin-airindia, those limits apply to each websocket connection.
